app/README.md

```diff
@@ -72,14 +72,17 @@ The filter `setid` is used in `/app/.gitattributes` so that it replaces your app
 
 1. Build your app by running `yarn build` in **/app/jenkins-for-jira-ui**.
 2. Deploy your app by running `forge deploy` in **/app**.
-3. Install your app to your Jira site by running `forge install` in **/app**.
+   1.   If everything is good except for 2 acceptable errors...`invalid value 'devops:deploymentInfoProvider' in modules`
```

Comment:
I think its better to address this in a different part. Maybe a sub heading or a small note saying _run forge deploy...._ if you face these 2 errors.